config/routes.rb

```diff
@@ -1,4 +1,10 @@
 Rails.application.routes.draw do
+  root 'static_pages#home'
+
+  get 'static_pages/help'
+
+  get 'static_pages/about'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.